### PR TITLE
Add support for two windings transformer tabular modifications.

### PR DESCRIPTION
**Pull request by flomillot**

### CODE REVIEW

**Comment by flomillot** [comment]
Waiting for a decision about unit conversion

**Review by klesaulnier — APPROVED** [review]

**Review by antoinebhs — COMMENTED** [review]
The filters in the AG Grid table behave as filters on string whereas here we would expect filtering on number ?
It seems to be OK in the **Spreadsheet** tab.
image
It seems to be incorrect for other modification types too...

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts [hunk 1]
```diff
@@ -26,12 +38,24 @@ export const TABULAR_MODIFICATION_FIELDS: TabularModificationFields = {
         'lowVoltageLimit',
         'highVoltageLimit',
     ],
+    TWO_WINDINGS_TRANSFORMER: [
+        EQUIPMENT_ID,
```
Comment (antoinebhs) [inline src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts:42]: Can't we also use constants for voltage_level, load and generator for consistency ?

File: src/translations/network-modifications-fr.json [hunk 2]
```diff
@@ -38,5 +38,6 @@
     "network_modifications/tabular/GENERATOR_MODIFICATION": "modifications de générateurs",
     "network_modifications/tabular/LOAD_MODIFICATION": "modifications de consommations",
     "network_modifications/BY_FORMULA_MODIFICATION": "Modification par formule {computedLabel}",
-    "network_modifications/tabular/VOLTAGE_LEVEL_MODIFICATION": "modifications de postes"
+    "network_modifications/tabular/VOLTAGE_LEVEL_MODIFICATION": "modifications de postes",
+    "network_modifications/tabular/TWO_WINDINGS_TRANSFORMER_MODIFICATION": "modifications de transformateurs à 2 enroulements"
```
Comment (antoinebhs) [inline src/translations/network-modifications-fr.json:42]: this is a very long label...
image

Comment (flomillot) [reply src/translations/network-modifications-fr.json:42]: It is, what do you suggest ?
For me the "modifications" at first is too much, but it have been decided like that.
Anyway it's a more global problem in all the app.

Comment (antoinebhs) [reply src/translations/network-modifications-fr.json:43]: I think we should do one iteration with a PO. I went for this for deletion by filter:
image
`($equipmentType)`

Comment (flomillot) [reply src/translations/network-modifications-fr.json:43]: Ok, it's better IMO and we should homogenize anyway

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts (hunk 1 above)
Comment (flomillot) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts:49]: We should indeed, I don't know why they didn't do it

Comment (flomillot) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts:49]: The thing is we can't "clean" the code in US PR, and some peoples are working on these US.
We should create a TS to clean all this IMO.

**Review by antoinebhs — APPROVED** [review]
Code OK
Tests OK, should homogenize the labels in another PR as well as the filter in AgGrid able/use of constants

**Pull request merged**